internal/pkg/cli/deploy/svc.go

```diff
@@ -1401,7 +1401,7 @@ func (d *lbWebSvcDeployer) validateALBRuntime() error {
 
 		cdnCert := d.environmentConfig.CDNConfig.Config.Certificate
 		albCertValidator := d.newAliasCertValidator(nil)
-		cfCertValidator := d.newAliasCertValidator(aws.String(envCloudFrontCertRegion))
+		cfCertValidator := d.newAliasCertValidator(aws.String(manifest.EnvCloudFrontCertRegion))
```

Comment:
It's just a bit awkward to read the CloudFront cert region from the `manifest` pkg ya know? the `manifest` pkg should be responsible for fields and functionality about `manifest.yml` files whereas this seems to be a concern related to CloudFront.

What do you think of creating just a `aws/cloudfront` pkg with a single constant in it 😂 ?

```go
package cloudfront

const (
  // CertRegion is the only AWS region accepted by CloudFront while attaching certificates to a distribution.
  CertRegion = "us-east-1"
)
```

Reply:
Yeah I moved it because of circular dependencies, but as funny as it is this actually seems like a good solution lol 👍